### ISSUE TITLE
Redshift table creation sortkey definition/validation mismatch

### ISSUE THREAD

**Issue by jar-no1** [awslabs/aws-data-wrangler#226]
**Describe the bug**
Once trying to use Redshift functions with table creation the validation for `sortkey` parameter is checking for list but the parameter is defined to be optional string.

**To Reproduce**
Use Redshift DB function ` copy_files_to_redshift(..., sortkey: Optional[str] = None, ...)`
See validation: https://github.com/awslabs/aws-data-wrangler/blob/1.1.1/awswrangler/db.py#L769-L773

Assuming the definition should be like on `primary_keys`
```
    sortkey : str, optional
        List of columns to be sorted.
    primary_keys : List[str], optional
        Primary keys.
```
so that the given list is stringified later on the function
R742:
`sortkey_str: str = f"\n{sortstyle} SORTKEY({','.join(sortkey)})" if sortkey else ""`

https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_NEW.html

**Comment by igorborgest** [comment]
Thanks @jar-no1, it makes sense!

We will release a new version `1.1.2` later today with this fix.

**Comment by igorborgest** [comment]
Available on version [1.1.2](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.1.2)
